### PR TITLE
removed references of sambaverse

### PR DESCRIPTION
**Pull request by snova-jorgep**

### CODE REVIEW

File: utils/guardrails/guard.py [hunk 1]
```diff
@@ -40,12 +38,10 @@ def __init__(
         Initialize Guard class with specified LLM and guardrails.
 
         Parameters:
-        - api (str): The LLM API to use, either 'sambaverse' or 'sambastudio'. Default is 'sambaverse'.
+        - api (str): The LLM API to use, Default is 'sambastudio'.
```
Comment (snova-rodrigom) [inline utils/guardrails/guard.py:41]: There's a small typo here in the comma

File: utils/guardrails/guard.py [hunk 2]
```diff
@@ -22,12 +22,10 @@ class Guard:
 
     def __init__(
         self,
-        api: str = 'sambaverse',
+        api: str = 'sambastudio',
```
Comment (snova-rodrigom) [inline utils/guardrails/guard.py:25]: Just to confirm, shouldn't the default here be sncloud?

File: utils/guardrails/guard.py [hunk 3]
```diff
@@ -189,7 +182,7 @@ def evaluate(
     parser = argparse.ArgumentParser()
     parser.add_argument('--message', type=str, help='message to check')
     parser.add_argument('--role', type=str, help='role of the message')
-    parser.add_argument('--api', default='sambaverse', type=str, help='sambaverse or sambastudio')
+    parser.add_argument('--api', default='sambastudio', type=str, help='api to use')
```
Comment (snova-rodrigom) [inline utils/guardrails/guard.py:185]: Just to confirm, shouldn't the default here be sncloud?

**Review by snova-rodrigom — COMMENTED** [review]
Thanks! Left some small comments

File: utils/guardrails/guard.py (hunk 2 above)
Comment (snova-jorgep) [reply utils/guardrails/guard.py:25]: no llamaguard is not available yet in sncloud

File: utils/guardrails/guard.py (hunk 3 above)
Comment (snova-jorgep) [reply utils/guardrails/guard.py:185]: no llamaguard is not available yet in sncloud

**Review by snova-luiss — APPROVED** [review]

**Pull request merged**